packages/@o3r/workspace/schematics/library/index.spec.ts

```diff
@@ -77,14 +118,16 @@ describe('New module generator', () => {
       runner.registerCollection('@schematics/angular', path.resolve(path.dirname(angularPackageJson), require(angularPackageJson).schematics));
       runner.registerCollection('@nx/workspace', path.resolve(path.dirname(nxWorkspacePackageJson), require(nxWorkspacePackageJson).generators));
       const tree = await runner.runExternalSchematic('schematics', 'library', {
-        path: 'packages-test',
         name: '@my/new-module',
         projectName: 'test-module-name',
         skipLinter: true
       }, initialTree);
 
       expect(tree.exists('/packages-test/my-new-module/project.json')).toBe(true);
-      expect(tree.readContent('/packages-test/my-new-module/project.json')).toContain('"name": "test-module-name"');
+      const projectJson: any = tree.readJson('/packages-test/my-new-module/project.json');
+      expect(projectJson.name).toBe('test-module-name');
+      expect(tree.exists(projectJson.targets.test.options.jestConfig)).toBe(true);
+      expect(projectJson.targets.test.executor).toBe('@nrwl/jest:jest');
```

Comment:
There will be a conflict with https://github.com/AmadeusITGroup/otter/pull/2717 depending on which one will be merged first
```suggestion
      expect(projectJson.targets.test.executor).toBe('@nx/jest:jest');
```